### PR TITLE
Add kubeweekly cron jobs manifest.

### PR DESCRIPTION
**Pull request by 2pai**
this pr addressed to #2

### CODE REVIEW

File: manifests/kubeweekly-jobs.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: kubeweekly-jobs
```
Comment (girikuncoro) [inline manifests/kubeweekly-jobs.yaml]: for clarity better to add `namespace: default` explicitely

File: manifests/kubeweekly-jobs.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+spec:
+  schedule: "*/12 * * * 3"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: kubeweekly
+            image: kubeweekly:latest
```
Comment (girikuncoro) [inline manifests/kubeweekly-jobs.yaml]: does this work? you need to specify your docker hub username if it stores there, i.e. `2pai/kubeweekly`
anyway, I have created org `cloudnativeid` in docker hub, lets store docker images there for now

File: manifests/kubeweekly-jobs.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: kubeweekly-jobs
+spec:
+  schedule: "*/12 * * * 3"
```
Comment (girikuncoro) [inline manifests/kubeweekly-jobs.yaml]: we will compile reading lists every weekend (bacaan akhir pekan)

File: manifests/kubeweekly-jobs.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: kubeweekly
+            image: kubeweekly:latest
+            args:
+            - npm run trigger
+          restartPolicy: OnFailure
```
Comment (girikuncoro) [inline manifests/kubeweekly-jobs.yaml]: it's better to add newline in the end

**Review by girikuncoro — CHANGES_REQUESTED** [review]

File: manifests/kubeweekly-jobs.yaml (hunk 3 above)
Comment (irvifa) [reply manifests/kubeweekly-jobs.yaml]: Probably add coments on what is this cron is doing

File: manifests/kubeweekly-jobs.yaml (hunk 2 above)
Comment (irvifa) [reply manifests/kubeweekly-jobs.yaml]: Agree with Giri better not using individual docker user.

**Comment by irvifa** [comment]
Can you also include the source code of your image in there @2pai ? Will be better if we're using a variable for each variable such as namespace and image so this `jobs.yaml` can be reused in the future by someone else who wants to use it.

File: manifests/kubeweekly-jobs.yaml (hunk 2 above)
Comment (2pai) [reply manifests/kubeweekly-jobs.yaml]: ok noted
i think i have to fix the docker images first , is that okay?

Comment (girikuncoro) [reply manifests/kubeweekly-jobs.yaml]: yeah you can just commit Dockerfile here

**Comment by 2pai** [comment]
> Can you also include the source code of your image in there @2pai ? Will be better if we're using a variable for each variable such as namespace and image so this `jobs.yaml` can be reused in the future by someone else who wants to use it.

the images source code is reference to this [repository](https://github.com/cloudnative-id/kubeweekly-scrapper)
but i have to add some enhancement , i will update it later
is that okay?

**Comment by girikuncoro** [comment]
sure take your time

**Comment by zufardhiyaulhaq** [comment]
@2pai so this scrapper will create YAML files that store inside the container? Maybe I can write some code to receive the YAML files and send them to telegram & twitter.

**Comment by irvifa** [comment]
@zufardhiyaulhaq the code for Iqbal code is on different repo, may be create a different repo for the telegram bot 😄

**Comment by irvifa** [comment]
@2pai related to this https://github.com/2pai/kubeops/pull/1
I create a helm char for the kubeweekly 🙇

File: manifests/kubeweekly-jobs.yaml (hunk 2 above)
Comment (2pai) [reply manifests/kubeweekly-jobs.yaml]: the images is already updated
https://hub.docker.com/r/cloudnativeid/kubeweekly

File: manifests/kubeweekly-jobs.yaml (hunk 3 above)
Comment (2pai) [reply manifests/kubeweekly-jobs.yaml]: addressed here https://github.com/cloudnative-id/kubeops/pull/3/commits/e8f6b264a502f199cea8f65e4b8c92bb2a70868f

**Comment by irvifa** [comment]
@2pai https://github.com/2pai/kubeops/pull/2 kindly review 🙇 kind of messed up with the templates 😓

File: manifests/kubeweekly-jobs.yaml (hunk 1 above)
Comment (2pai) [reply manifests/kubeweekly-jobs.yaml]: addressed here https://github.com/cloudnative-id/kubeops/pull/3/commits/f08dc065397332554b19424289a1ca70a06d156d

File: manifests/kubeweekly-jobs.yaml (hunk 4 above)
Comment (2pai) [reply manifests/kubeweekly-jobs.yaml]: https://github.com/cloudnative-id/kubeops/pull/3/commits/f08dc065397332554b19424289a1ca70a06d156d

**Pull request merged**